Service/src/main/java/dao/UserDAO.java

```diff
@@ -0,0 +1,49 @@
+package dao;
+
+import model.users.Client;
+import java.sql.*;
+
+public class UserDAO {
+    private final String url = "jdbc:postgresql://localhost:5432/my_ticket_service_db";
+    private final String user = "postgres";
+    private final String password = "1111";
+
+    public void saveUser(Client client){
+        String sql = "INSERT INTO \"User\"  (name, creation_date) VALUES (?, ?)";
+        try(Connection conn = DriverManager.getConnection(url, user, password); PreparedStatement pstmt = conn.prepareStatement(sql)){
+            pstmt.setString(1, client.getName());
+            pstmt.setDate(2, Date.valueOf(client.getCreationDate()));
+            pstmt.executeUpdate();
+        }catch(SQLException ex){
+            System.out.println("Cannot update the database");
+            ex.printStackTrace();
+
+        }
+    }
+
+    public Client fetchUserById(int id){
+        String sql = "SELECT * FROM \"User\" WHERE id = ?";
+        try(Connection conn = DriverManager.getConnection(url, user,password); PreparedStatement pstmt = conn.prepareStatement(sql)){
+            pstmt.setInt(1, id);
+            ResultSet rs = pstmt.executeQuery();
+            if(rs.next()){
+                return new Client(rs.getString("name"), rs.getInt("id"), rs.getDate("creation_date").toLocalDate());
+            }
+        }catch(SQLException ex){
+            System.out.println("Cannot fetch user by this ID");
+        }
+        return null;
+    }
+
+    public void deleteUserById(int id)  {
+        String sql = "DELETE FROM \"User\" WHERE id = ?";
```

Comment:
what about deleting tickets which related to user?

Reply:
It is operation with tickets. I thought that it will be logic to realize this opportunity in TicketDao.

Reply:
I mean when we're deleting user we need also to delete all tickets which are related to deleted user(in order not to have the tickets containing not existing userId)

How is this configured in your app?